project-koku/koku
# [COST-4333] - Adding dtypes for Azure

## Jira Ticket

[COST-4333](https://issues.redhat.com/browse/COST-4333)

## Description

This change will commonize dtype handling and correctly set the csv converters for Azure

## Testing

1. Checkout Branch
2. Run koku with updated trino/hive versions
3.  create/load data for Azure provider
4. see data is processed correctly

## Notes
trino diff
deploy/trino/etc/catalog/hive.properties
 
-parquet.optimized-reader.enabled=false
+# parquet.optimized-reader.enabled=false

diff --git a/deploy/trino/etc/jvm.config b/deploy/trino/etc/jvm.config

+-Dfile.encoding=UTF-8

diff --git a/docker-compose.arm.yml b/docker-compose.arm.yml

   hive-metastore:
-    image: quay.io/mskarbek/ubi-hive:3.1.2-metastore-008
+    image: quay.io/mskarbek/ubi-hive:3.1.3-metastore-024
 
   trino:
-    image: quay.io/mskarbek/ubi-trino:411-001
+    image: quay.io/mskarbek/ubi-trino:427-001

diff --git a/docker-compose.yml b/docker-compose.yml
   hive-metastore:
     container_name: hive-metastore
-    image: quay.io/cloudservices/ubi-hive:3.1.2-metastore-008
+    image: quay.io/cloudservices/ubi-hive:latest

   trino:
     container_name: trino
-    image: quay.io/cloudservices/ubi-trino:411-002
+    image: quay.io/cloudservices/ubi-trino:latest
...
